scripts/deps-ppa/static_z3.sh

```diff
@@ -40,7 +40,7 @@ pparepo=cpp-build-deps
 ppafilesurl=https://launchpad.net/~ethereum/+archive/ubuntu/${pparepo}/+files
 
 # Fetch source
-git clone --depth 1 --branch Z3-${version} https://github.com/Z3Prover/z3.git
+git clone --depth 1 --branch z3-${version} https://github.com/Z3Prover/z3.git
```

Comment:
Seems like it was a fluke for 4.8.5 that the tag was upper case and it is usually lower case.

Reply:
Yea, had notice that as well

Reply:
Interesting, just tagging this as relevant for clang docker build as well if an update there is necessary

Reply:
Yep, changed to lower-case there as well now.